tests/mock_update_catalog.py

```diff
@@ -0,0 +1,120 @@
+# MIT License
+#
+# (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP

```

Comment:
This is a new file, so the copyright header should only contain the current year.
```suggestion
# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
```